graders/python/python_autograder/pltest.py

```diff
@@ -0,0 +1,127 @@
+from unittest import TestLoader, TestSuite
+import json
+import traceback
+import os
+from collections import defaultdict
+from pl_result import PrairieTestResult
+
+
+"""
+The main python entrypoint for the autograder framework.
+Loads and executes test cases.
+"""
```

Comment:
This file is called `pltest` whereas all the other ones have a `pl_` prefix (with underscore). Should we standardize?

Reply:
@mwest1066 this naming mirrors the existing `pltest` file across AGs: `Python`, `Java`, `C++`, and `R`.

Reply:
But this is the only AG that we actually have included in PL?

Reply:
@mwest1066 true, the Python autograder is the only one included in PL core. I suppose why I bring this up is that the other AGs were derived from the present AG. 

[Current `pltest.py` AG.](https://github.com/PrairieLearn/PrairieLearn/blob/905d6d62afb11f6b187653aea7a178b0aeb58844/exampleCourse/serverFilesCourse/python_autograder/pltest.py)

Reply:
I'll change it, the lack of underscore is bothersome